docs/development/extensions-core/kinesis-ingestion.md

```diff
@@ -23,154 +23,259 @@ sidebar_label: "Amazon Kinesis"
 
-## Submitting a Supervisor Spec
+## Supervisor spec
 
-To use the Kinesis indexing service, load the `druid-kinesis-indexing-service` extension on both the Overlord and the MiddleManagers. Druid starts a supervisor for a dataSource when you submit a supervisor spec. Submit your supervisor spec to the following endpoint:
+The following table outlines the high-level configuration options for the Kinesis supervisor object. 
+See [Supervisor API](../../api-reference/supervisor-api.md) for more information.
 
-`http://<OVERLORD_IP>:<OVERLORD_PORT>/druid/indexer/v1/supervisor`
+|Property|Type|Description|Required|
+|--------|----|-----------|--------|
+|`type`|String|The supervisor type; this should always be `kinesis`.|Yes|
+|`spec`|Object|The container object for the supervisor configuration.|Yes|
+|`ioConfig`|Object|The [I/O configuration](#supervisor-io-configuration) object for configuring Kafka connection and I/O-related settings for the supervisor and indexing task.|Yes|
+|`dataSchema`|Object|The schema used by the Kinesis indexing task during ingestion. See [`dataSchema`](../../ingestion/ingestion-spec.md#dataschema) for more information.|Yes|
+|`tuningConfig`|Object|The [tuning configuration](#supervisor-tuning-configuration) object for configuring performance-related settings for the supervisor and indexing tasks.|No|
 
-For example:
+Druid starts a new supervisor when you define a supervisor spec.
+To create a supervisor, send a `POST` request to the `/druid/indexer/v1/supervisor` endpoint.
+Once created, the supervisor persists in the configured metadata database. There can only be a single supervisor per datasource, and submitting a second spec for the same datasource overwrites the previous one.
 
-```sh
-curl -X POST -H 'Content-Type: application/json' -d @supervisor-spec.json http://localhost:8090/druid/indexer/v1/supervisor
-```
+When an Overlord gains leadership, either by being started or as a result of another Overlord failing, it spawns
+a supervisor for each supervisor spec in the metadata database. The supervisor then discovers running Kinesis indexing
+tasks and attempts to adopt them if they are compatible with the supervisor's configuration. If they are not
+compatible because they have a different ingestion spec or shard allocation, the tasks are killed and the
+supervisor creates a new set of tasks. In this way, the supervisors persist across Overlord restarts and failovers.
 
-Where the file `supervisor-spec.json` contains a Kinesis supervisor spec:
+The following example shows how to submit a supervisor spec for a stream with the name `KinesisStream`.
+In this example, `http://SERVICE_IP:SERVICE_PORT` is a placeholder for the server address of deployment and the service port.
 
-```json
-{
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl -X POST "http://SERVICE_IP:SERVICE_PORT/druid/indexer/v1/supervisor" \
```

Comment:
normally we're running all HTTP API calls through the router. Retest it with this :
`curl -X POST "http://SERVICE_IP:SERVICE_PORT/druid/router/v1/supervisor"`

`